pool/src/testutils.rs

```diff
@@ -170,8 +170,8 @@ pub(crate) fn create_comet_lp_pool<'a>(
     );
 
     client.set_swap_fee(&0_0030000, &admin);
-    client.finalize();
     client.set_public_swap(&admin, &true);
+    client.finalize();
```

Comment:
why was this change necessary?

IIRC `finalize` had nothing to do with `set_public_swap`, but this could have changed on comet's end?